tests/tests/tier0/bluechi-agent-logisquiet/test_agent_logisquiet.py

```diff
@@ -0,0 +1,80 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+import pytest
+from typing import Dict
+
+from bluechi_test.test import BluechiTest
+from bluechi_test.container import BluechiControllerContainer, BluechiNodeContainer
+from bluechi_test.config import BluechiControllerConfig, BluechiNodeConfig
+
+NODE_GOOD = "node-good"
+NODE_WITH_LONG_LOGISQUIET = "node-long-logisquiet"
+NODE_WITH_NOT_VALID_VALUE = "node-with-not-valid-value"
+NODE_WITH_NUMBERS_ONLY_IN_LOGISQUIET = "node-numbers-only"
+
+
+def start_with_invalid_port(ctrl: BluechiControllerContainer, nodes: Dict[str, BluechiNodeContainer]):
+    _, output = nodes[NODE_GOOD].exec_run('systemctl status bluechi-agent')
+    output_systemd = str(output)
+    if "fail" in output_systemd.lower():
+        raise Exception(f"{NODE_GOOD} bluechi-agent should NOT failed during the start of the service")
+
+    _, output = nodes[NODE_WITH_LONG_LOGISQUIET].exec_run('systemctl status bluechi-agent')
+    output_systemd = str(output)
+    if "fail" not in output_systemd.lower():
+        raise Exception(
+                f"{NODE_WITH_LONG_LOGISQUIET} bluechi-agent should FAIL "
+                "during the start of the service"
+        )
+
+    _, output = nodes[NODE_WITH_NOT_VALID_VALUE].exec_run('systemctl status bluechi-agent')
+    output_systemd = str(output)
+    if "fail" in output_systemd.lower():
+        raise Exception(
+                f"{NODE_WITH_NOT_VALID_VALUE} bluechi-agent should "
+                "NOT failed during the start of the service")
+
+    _, output = nodes[NODE_WITH_NUMBERS_ONLY_IN_LOGISQUIET].exec_run('systemctl status bluechi-agent')
+    output_systemd = str(output)
+    if "fail" in output_systemd.lower():
+        raise Exception(
+                f"{NODE_WITH_NUMBERS_ONLY_IN_LOGISQUIET} bluechi-agent should "
+                "NOT failed during the start of the service")
```

Comment:
Optional:
You could also check if the nodes are shown as on-/offline by using the python bindings (see example [here](https://github.com/containers/bluechi/tree/main/tests/tests/tier0/monitor-open-close)).